R/data.R

```diff
@@ -69,17 +69,17 @@
 #'  \item{percasian}{Percent Asian.}
 #'  \item{percother}{Percent other races.}
 #'  \item{popadults}{Number of adults.}
-#'  \item{perchsd}{}
+#'  \item{perchsd}{Percent with high school diploma.}
 #'  \item{percollege}{Percent college educated.}
-#'  \item{percprof}{Percent profession.}
-#'  \item{poppovertyknown}{}
-#'  \item{percpovertyknown}{}
-#'  \item{percbelowpoverty}{}
-#'  \item{percchildbelowpovert}{}
-#'  \item{percadultpoverty}{}
-#'  \item{percelderlypoverty}{}
-#'  \item{inmetro}{In a metro area.}
-#'  \item{category}{}
+#'  \item{percprof}{Percent with professional degree.}
```

Comment:
Got the same hint [from that one homework assignment](https://www.coursehero.com/file/26646690/JohnLonaipdf/). And with the context of the other variables (`perchsd` and `percollege`), this would make more sense that this is a professional degree rather than merely "profession". The percentages of professional degrees in the data set are much lower than the `percollege` variable, which is a good sanity check.